src/components/MultiHopTrade/components/TradeInput/components/Claim/Claim.tsx

```diff
@@ -79,9 +61,15 @@ export const ClaimRoutes: React.FC = () => {
   }, [activeClaim, claimTxHash, claimTxStatus])
 
   return (
-    <AnimatePresence mode='wait' initial={false}>
+    <MemoryRouter initialEntries={ClaimRouteEntries} initialIndex={0}>
       <Switch location={location}>
-        <Suspense>
```

Comment:
q: can we keep wrapping `<Route />` in suspense with refactored routing?

Reply:
no suspense because its at the page level - removed it because it would cause glitching and layout shifting when opening the bridge claim